<ui version="4.0" >
 <author>Walter Roth</author>
 <comment>This is the basic log dialog, which is shown 
during communication with the bank server.
This log dialog shows the results of the 
communication</comment>
 <class>GuiLog</class>
 <widget class="QWidget" name="GuiLog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Log Messages:</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEditLog" />
         </item>
         <item>
          <widget class="QPushButton" name="buttonHideDetails" >
           <property name="text" >
            <string>Hide &amp;Details</string>
           </property>
           <property name="shortcut" >
            <string>Alt+D</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1" >
           <property name="text" >
            <string>Executing job queue. Please wait while encrypted data are transmitted to your bank through the internet.</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPresentJob" >
           <property name="text" >
            <string/>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDetails" >
           <property name="text" >
            <string>Show &amp;Details</string>
           </property>
           <property name="shortcut" >
            <string>Alt+D</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonClose" >
     <property name="text" >
      <string>&amp;Close</string>
     </property>
     <property name="shortcut" >
      <string>Alt+C</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>buttonDetails</sender>
   <signal>pressed()</signal>
   <receiver>GuiLog</receiver>
   <slot>slotShowDetails()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonHideDetails</sender>
   <signal>pressed()</signal>
   <receiver>GuiLog</receiver>
   <slot>slotHideDetails()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>GuiLog</receiver>
   <slot>slotCloseRequest()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
